File: features/app/app/Ver.go
package app

import (
	"github.com/hailongz/golang/db"
)

type Ver struct {
	db.Object
	Appid	int64	`json:"appid" name:"appid" title:"用户ID" index:"ASC"`
	Ver	int32	`json:"ver" name:"ver" title:"版本号" index:"DESC"`
	Info	interface{}	`json:"info,omitempty" name:"info" title:"应用信息 JSON" length:"-1"`
	Options	interface{}	`json:"options,omitempty" name:"options" title:"其他数据" length:"-1"`
	Ctime	int64	`json:"ctime" name:"ctime" title:"创建时间"`
}

func (O *Ver) GetName() string {
	return "ver"
}

func (O *Ver) GetTitle() string {
	return "Ver"
}

File: features/app/ts/Ver.ts
import { int64, int32 } from "./lib/less";

/**
 * Ver
 * @type db
 */
export class Ver {

    /**
    * ID
    */
    id: int64 = 0

    /**
    * 用户ID
    * @index ASC
    */
    appid: int64 = 0

    /**
     * 版本号
     * @index DESC
     */
    ver: int32 = 0

    /**
     * 应用信息 JSON
     * @length -1
     */
    info: any

    /**
     * 其他数据
     * @length -1
     */
    options: any

    /**
     * 创建时间
     */
    ctime: int64 = 0

}